### PR TITLE
Show only one calendar

### PR DESCRIPTION
**Pull request by LeticiaMachadoo**

### CODE REVIEW

File: src/DateRangePicker/DateRangePicker.tsx [hunk 1]
```diff
@@ -520,10 +531,16 @@ class DateRangePicker extends React.Component<DateRangePickerProps, DateRangePic
             <div className={this.addPrefix('daterange-header')}>
               {this.getDateString(selectValue as ValueType)}
             </div>
-            <div className={this.addPrefix('daterange-calendar-group')}>
-              <DatePicker index={0} {...pickerProps} />
-              <DatePicker index={1} {...pickerProps} />
-            </div>
+            {showOneCalendar ? (
```
Comment (simonguo) [inline src/DateRangePicker/DateRangePicker.tsx]: Missing test case for `showOneCalendar` property.

**Comment by simonguo** [comment]
@LeticiaMachadoo You can run unit tests with:
```
M=DateRangePicker npm run tdd
```
And I hope you can add unit tests for new features.

**Comment by simonguo** [comment]
@LeticiaMachadoo  
Thank you for your contribution, we will merge this PR in a recent new version.

**Comment by boriscosic** [comment]
@LeticiaMachadoo thanks for this PR. I've noticed a small issue with next button in single calendar which stays disabled. It should be enabled as per default mode. 

Screen Shot 2020-03-11 at 9 26 56 AM
Screen Shot 2020-03-11 at 9 25 42 AM

**Comment by ehsantekgenisys** [comment]
Guys any timeline for release of this feature?

**Comment by LeticiaMachadoo** [comment]
> @LeticiaMachadoo thanks for this PR. I've noticed a small issue with next button in single calendar which stays disabled. It should be enabled as per default mode.
> 
> Screen Shot 2020-03-11 at 9 26 56 AM
> Screen Shot 2020-03-11 at 9 25 42 AM
> 
> (edit)
> It looks like the issue is here:
> 
> https://github.com/rsuite/rsuite/blob/489f765345c5e1335470e2b057c6e33ebb89c29b/src/DateRangePicker/DateRangePicker.tsx#L510-L538
> 
> `showOneCalendar` needs to be passed into the calendar props and down so it is correctly picked up in `disabledForword` in Calendar.tsx.
> 
> This is a super useful feature for mobile as the DaterangePicker is currently off the screen.

Solved

**Comment by simonguo** [comment]
Published `rsuite@4.3.2`

https://rsuitejs.com/en/components/date-range-picker

**Pull request merged**